src/Deserializers/SnakListDeserializer.php

```diff
@@ -52,6 +54,10 @@ private function getDeserialized( array $serialization ) {
 
 		foreach ( $serialization as $key => $snakArray ) {
 			if ( is_string( $key ) ) {
+				if ( !is_array( $snakArray ) ) {
+					throw new DeserializationException( 'The snaks per property should be an array' );
```

Comment:
Both cases are old exception messages, and I did not wanted to change them, but I can.